### ISSUE TITLE
Flatpak apps

### ISSUE THREAD

**Issue by Donkere-vader** [b1izzard-34/Skeuo-icon-theme#8]
You have beautifull icons for spotify and discord. But not for the flatpak versions. It would be nice if they werer supported as well.

**Comment by maydayv7** [comment]
Icon names for flatpak:
Spotify- com.spotify.Client
Discord- com.discordapp.Discord

**Comment by TheCultLeader666** [comment]
It seems like flatpak has standard app icons rather than just using whatever the icon name is chosen to be by the developer, but I could be wrong

**Comment by b1izzard-34** [comment]
yea I don't use flatpak so got no idea about how it works. If anyone has any idea then just lemme know. @maydayv7 if you use flatpaks then can you try renaming the icon of spotify to com.spotify.Client and check if it reflects for flatpaks?

**Comment by maydayv7** [comment]
Yes, for most flatpak apps, a different name is required. The way Papirus solves this problem is by using symlinks.
